internal/binarylog/binarylog.go

```diff
@@ -98,7 +98,7 @@ func (l *logger) setDefaultMethodLogger(ml *methodLoggerConfig) error {
 // New methodLogger with same service overrides the old one.
 func (l *logger) setServiceMethodLogger(service string, ml *methodLoggerConfig) error {
 	if _, ok := l.services[service]; ok {
-		return fmt.Errorf("conflicting rules for service %v found", service)
+		return fmt.Errorf("conflicting service rules for service %v found", service)
 	}
```

Comment:
The function level comment says the following:
`New methodLogger with same service overrides the old one.`

So, ideally we shouldn't even be checking whether a methodLogger for the provided service already exists in the map or not. We should be blindly setting the map entry.

Did you find these log messages to be useful/not-useful at any point?
Did you run into these errors?

Reply:
> So, ideally we shouldn't even be checking whether a methodLogger for the provided service already exists in the map or not. We should be blindly setting the map entry.

According to the function doc, yes, you're right. Nothing needs to check just blindly set it should be ok. We should clean this logic.

> Did you find these log messages to be useful/not-useful at any point?
Did you run into these errors?

I have not run into errors about these log messages. But adding the name for the map in the log messages will help to diagnose problems quickly and clearly.

Reply:
/cc @easwars @dfawley